### ISSUE TITLE
wrecon v1.04 not starting

### ISSUE THREAD

**Issue by reddy75** [reddy75/wrecon#3]
Script not starting due to following error

```
python: wrong arguments for function "register" (script: -)
python: stdout/stderr (?): TypeError: function takes exactly 7 arguments (6 given)
python: stdout/stderr (?): The above exception was the direct cause of the following exception:
python: stdout/stderr (?): Traceback (most recent call last):
python: stdout/stderr (?):   File "/home/reddy/.weechat/python/autoload/wrecon.py", line 148, in <module>
python: stdout/stderr (?):     weepython: wrong arguments for function "register" (script: -)
python: stdout/stderr (?): TypeError: function takes exactly 7 arguments (6 given)
python: stdout/stderr (?): The above exception was the direct cause of the following exception:
python: stdout/stderr (?): Traceback (most recent call last):
python: stdout/stderr (?):   File "/home/reddy/.weechat/python/autoload/wrecon.py", line 148, in <module>
python: stdout/stderr (?):     weechat.register(SCRIPT_NAME, SCRIPT_VERSION, SCRIPT_LICENSE, SCRIPT_DESC, SCRIPT_UNLOAD, 'UTF-8')
python: stdout/stderr (?): SystemError: <built-in function register> returned a result with an error set
python: unable to parse file "/home/reddy/.weechat/python/autoload/wrecon.py"chat.register(SCRIPT_NAME, SCRIPT_VERSION, SCRIPT_LICENSE, SCRIPT_DESC, SCRIPT_UNLOAD, 'UTF-8')
python: stdout/stderr (?): SystemError: <built-in function register> returned a result with an error set
python: unable to parse file "/home/reddy/.weechat/python/autoload/wrecon.py"
```

**Comment by reddy75** [comment]
Fix will be released in version v1.05

**Comment by reddy75** [comment]
Issue has been fixed in released v1.05
